js/widgets/toolbar.js

```diff
@@ -47,7 +48,7 @@ return $.widget( "mobile.toolbar", {
 
 	_create: function() {
 		var leftbutton, rightbutton,
-			role = this.element.is( ":jqmData(role='header')" ) ? "header" : "footer",
+			role =  this.element.is( ":jqmData(role='header')" ) ? "header" : "footer",
```

Comment:
@arschmitz I feel like contentRole is a little ambiguous. In my mind it might be more intuitive to do `toolbarLocation` or `toolbarType`